pegen/__init__.py

```diff
@@ -10,7 +11,6 @@
 __all__ = [
```

Comment:
Maybe we should make `__init__.py` totally empty? The way it is now you can't run a parser without importing all the generator code.